### PR TITLE
Add couette test, analytic solution, and example

### LINKED ISSUE #135: Couette flow analytic solution and boundary
- add a couette flow example
- add an analytic_solution
- update boundary condition for top and bottom plates (currently too low at `< 1e-6`)

### PR DESCRIPTION
**Pull request by PhiSpel**
closes  #135

- [x] add a couette flow example
- [x] add an analytic_solution
- [x] update boundary condition for top and bottom plates (currently too low at `< 1e-6`)
- [x] while I was at it, I added a Poiseuille example

### CODE REVIEW

File: lettuce/flows/couette.py [hunk 1]
```diff
@@ -11,20 +11,28 @@
...
+        x, y = self.grid
+        u = np.array([dvdy*y + v0], dtype=float)
+        return u
 
     def initial_solution(self, x):
         return np.array([0 * x[0]], dtype=float), np.array([0 * x[0], 0 * x[1]], dtype=float)
 
+    @property
+    def mask(self):
+        return self._mask\
```
Comment (VascoSch92) [inline lettuce/flows/couette.py:34]:
```suggestion
        return self._mask
```
or it was correct?

Comment (PhiSpel) [reply lettuce/flows/couette.py]: It does not change the code's behavior but it's a good suggestion

File: lettuce/flows/couette.py [hunk 2]
```diff
@@ -11,20 +11,28 @@
 class CouetteFlow2D(object):
     def __init__(self, resolution, reynolds_number, mach_number, lattice):
         self.resolution = resolution
+        self.shape = (resolution, resolution)
+        self._mask = np.zeros(shape=self.shape, dtype=bool)
```
Comment (VascoSch92) [inline lettuce/flows/couette.py:15]: you are using self.shape only in self._mask. Perhaps delete this variable ;-)

Comment (PhiSpel) [reply lettuce/flows/couette.py:15]: @VascoSch92 sorry for the late reply and thanks for the suggestion. shape is a inherent field of any flow object as it can be important for post-processing. We are planning to integrate more generic classes such as the Flow class, so that such things will be clearer.

**Comment by PhiSpel** [comment]
continuing this in #163

**Comment by PhiSpel** [comment]
continuing this in #163
